### PR TITLE
[ADD] New module sale_mrp_bom

### PR DESCRIPTION
**Pull request by renatonlima**
This module allows define one BOM in sale order line.

### CODE REVIEW

**Comment by renatonlima** [comment]
Travis status is red in current branch 12.0, I made the PR #1229 to fix travis status, after merge 1229, just do a rebase in this PR to make this PR status green

**Review by HaraldPanten — APPROVED** [review]
As you need to activate a new security group, IMO that should be mentioned in the README file.

Except for this, overall LGTM

File: sale_mrp_bom/security/security.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="sale_mrp_bom_group" model="res.groups">
+        <field name="name">Allows definie BOM on sale order line</field>
```
Comment (HaraldPanten) [inline sale_mrp_bom/security/security.xml]:
```suggestion
        <field name="name">Allows to define a BOM on sale order lines</field>
```
Sounds better

File: sale_mrp_bom/README.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_mrp_bom
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/167/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows you to define a BOM in the sales order line.
```
Comment (rousseldenis) [inline sale_mrp_bom/README.rst]: Could you explain a little bit more what happen when sale order is confirmed ?

**Review by hparfr — APPROVED** [review]
Functional test: work as expected.
Thanks

**Review by marcelsavegnago — APPROVED** [review]

File: sale_mrp_bom/security/security.xml (hunk 1 above)
Comment (renatonlima) [reply sale_mrp_bom/security/security.xml]: @HaraldPanten thanks for your review, Done!

File: sale_mrp_bom/README.rst (hunk 2 above)
Comment (renatonlima) [reply sale_mrp_bom/README.rst]: Hello @rousseldenis thanks for your review, Done!

**Review by rvalyi — APPROVED** [review]
Approved. Also the request for more README details has been attended.

**Pull request merged**